h/traversal/roots.py

```diff
@@ -283,7 +285,12 @@ def __init__(self, request):
         self.user_svc = self.request.find_service(name="user")
 
     def __getitem__(self, userid):
-        user = self.user_svc.fetch(userid)
+        try:
+            user = self.user_svc.fetch(userid)
+        except InvalidUserId as e:
+            # In this context we failed because the user provided a bad id
+            # We need to tell them that so they can attempt to resolve it
```

Comment:
I don't think we need this comment—a 400 response communicates just that on its own :)

Reply:
Fair enough. I suppose I was trying to say why it was a 400.